test/e2e/bootstrap_resources.py

```diff
@@ -18,12 +18,13 @@
 from dataclasses import dataclass
 from acktest.bootstrapping import Resources
 from e2e import bootstrap_directory
+from e2e.bootstrappable.snapshots import Snapshots
 
 @dataclass
 class BootstrapResources(Resources):
 
-    def replacement_dict(self):
-        return {}
+    Snapshot1: Snapshots
+    Snapshot2: Snapshots
```

Comment:
Is there a reason we need two clusters? Are we able to run both tests on a single cluster? It would certainly save us some time when bootstrapping.